bridge/engine/ActiveMediaList.cpp

```diff
@@ -538,16 +536,13 @@ void ActiveMediaList::process(const uint64_t timestamp,
         outAudioMapChanged |= updateActiveAudioList(top.participant);
         auto const& curParticipant = _audioParticipants.find(top.participant);
 
-        if (top.score - top.noiseLevel > _activeTalkerSilenceThresholdDb)
+        if (top.score > _activeTalkerSilenceThresholdDb && activeTalkersSnapshot.count < activeTalkersSnapshot.maxSize)
```

Comment:
score is recent - noise , so I am not sure what this active talker selection was menat to be. One neede to be 18dB above 2x noise level to get into list.